judgels-backends/judgels-commons/judgels-persistence-core/src/main/java/judgels/persistence/hibernate/JudgelsHibernateDao.java

```diff
@@ -35,6 +35,12 @@ public M insert(M model) {
         return super.insert(model);
     }
 
+    @Override
+    public M insertWithJid(String jid, M model) {
```

Comment:
Can we add some test in `JudgelsHibernateDaoIntegrationTests`?

Reply:
sure!